_includes/home-page.html

```diff
@@ -234,8 +234,15 @@ <h1 class="used-by">Used By</h1>
 								<img src="./assets/images/company-logo/book-my-show-logo.png" alt="BookMyShow" loading="lazy"/>
 							</div>
 							<div class="company-image">
-								<a href="https://layer5.io/" target="_blank">
-									<img id="layer5-logo" src="./assets/images/company-logo/layer5-no-trim.svg" alt="Layer5" loading="lazy" />
+								<a href="https://layer5.io/"
+								 target="_blank">
+								<img src="./assets/images/company-logo/layer5-no-trim.svg"
+								id="logo-dark-light"
+								data-logo-for-dark="./assets/images/company-logo/layer5-light-no-trim.svg" 
+								data-logo-for-light="./assets/images/company-logo/layer5-no-trim.svg"
```

Comment:
file name should be consistent, in dark mode rename it like `layer5-dark-no-trim.svg` and vice versa for light mode
```suggestion
								data-logo-for-light="./assets/images/company-logo/layer5-light-no-trim.svg"
```